doc/source/serve/advanced.rst

```diff
@@ -269,3 +269,63 @@ Once a endpoint is deleted, its tag can be reused.
 .. code-block:: python
 
   serve.delete_endpoint("simple_endpoint")
+
+How can I monitor Serve?
```

Comment:
I'd probably just call this `Monitoring`

Reply:
+1